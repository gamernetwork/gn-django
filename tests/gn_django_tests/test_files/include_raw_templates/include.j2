<h1><code>include_raw</code> test</h1>

<p class="red">This text is red</p>

<style>
{% include_raw 'styles.css' %}
</style>

<div id="some-id">This text is not</div>
