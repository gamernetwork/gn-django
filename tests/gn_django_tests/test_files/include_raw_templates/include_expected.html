<h1><code>include_raw</code> test</h1>

<p class="red">This text is red</p>

<style>
body {
  background: papayawhip;
}

.red {
  color: red;
}

#some-id {
  color: green;
}

/* This is to make sure the template isn't being parsed by Jinja2 - it would
   throw an error due to the '{#' appearing to be an unclosed comment */
@media only screen and (max-width: 1260px) {#some-id{color: rebeccapurple;}}

</style>

<div id="some-id">This text is not</div>
